### ISSUE TITLE
[HW Accel Support]: Frigate doesn't start on Intel N100 

### ISSUE THREAD

**Issue by AlekseiLitvin** [blakeblackshear/frigate#7467]
### Describe the problem you are having

I run Frigate in the Docker container, on a mini PC based on an Intel N100 CPU.
Apparently, `the hwaccel_args` are not compatible with N100 CPU. I added an env variable  LIBVA_DRIVER_NAME: "i965", also tried `hwaccel_args: preset-intel-qsv-h264`, still no luck
What `hwaccel_args` should I use to make it work?

### Version

0.12.1-367d724

### Frigate config file

```yaml
mqtt:
  enabled: true
  host: <>
  port: <>
  topic_prefix: <>
  client_id: <>

cameras:
  front_yard: # <------ Name the camera
    ffmpeg:
      inputs:
        - path: <>
          roles:
            - record
        - path: <>
          roles:
            - detect 
      **hwaccel_args: preset-vaapi**
    detect: 
      enabled: True # <---- disable detection until you have a working camera feed
      width: 640 # <---- update for your camera's resolution
      height: 360 # <---- update for your camera's resolution
      fps: 10
    record:
        enabled: True
        events:
          retain:
            default: 1
    snapshots:
      enabled: True
      retain:
        default: 10
    mqtt:
      enabled: True
```

### docker-compose file or Docker CLI command

```yaml
frigate:
    depends_on:
      - mosquitto
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: ghcr.io/blakeblackshear/frigate:stable
    shm_size: "64mb" # update for your cameras based on calculation above
    devices:
      - /dev/bus/usb:/dev/bus/usb # passes the USB Coral, needs to be modified for other versions
      - /dev/apex_0:/dev/apex_0 # passes a PCIe Coral, follow driver instructions here https://coral.ai/docs/m2/get-started/#2a-on-linux
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /home/styrn/sh/frigate/config.yml:/config/config.yml
      - /home/styrn/sh/frigate/storage:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5000:5000"
      - "8554:8554" # RTSP feeds
      - "8555:8555/tcp" # WebRTC over tcp
      - "8555:8555/udp" # WebRTC over udp
    environment:
      FRIGATE_RTSP_PASSWORD: ${FRIGATE_PASSWORD}
      LIBVA_DRIVER_NAME: "i965"
```

### Relevant log output

```shell
2023-08-14 02:25:14.138039900  [INFO] Starting Frigate...
2023-08-14 02:25:15.181742842  [2023-08-14 02:25:15] frigate.app                    INFO    : Starting Frigate (0.12.1-367d724)
2023-08-14 02:25:15.201507871  [2023-08-14 02:25:15] peewee_migrate                 INFO    : Starting migrations
2023-08-14 02:25:15.205195694  [2023-08-14 02:25:15] peewee_migrate                 INFO    : There is nothing to migrate
2023-08-14 02:25:15.220640827  [2023-08-14 02:25:15] frigate.app                    INFO    : Output process started: 291
2023-08-14 02:25:15.220977836  [2023-08-14 02:25:15] detector.cpu                   INFO    : Starting detection process: 289
2023-08-14 02:25:15.221147269  [2023-08-14 02:25:15] frigate.detectors              WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
2023-08-14 02:25:15.226169629  [2023-08-14 02:25:15] frigate.app                    INFO    : Camera processor started for front_yard: 293
2023-08-14 02:25:15.230826337  [2023-08-14 02:25:15] frigate.app                    INFO    : Camera processor started for front_door: 294
2023-08-14 02:25:15.235014712  [2023-08-14 02:25:15] frigate.app                    INFO    : Capture process started for front_yard: 296
2023-08-14 02:25:15.238673890  [2023-08-14 02:25:15] frigate.app                    INFO    : Capture process started for front_door: 300
2023-08-14 02:25:16.977430670  [2023-08-14 02:25:16] frigate.video                  ERROR   : front_door: Unable to read frames from ffmpeg process.
2023-08-14 02:25:16.978078850  [2023-08-14 02:25:16] frigate.video                  ERROR   : front_door: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:25:17.688183106  [2023-08-14 02:25:17] frigate.video                  ERROR   : front_yard: Unable to read frames from ffmpeg process.
2023-08-14 02:25:17.688347935  [2023-08-14 02:25:17] frigate.video                  ERROR   : front_yard: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:25:35.290009869  [2023-08-14 02:25:35] watchdog.front_door            ERROR   : Ffmpeg process crashed unexpectedly for front_door.
2023-08-14 02:25:35.295273911  [2023-08-14 02:25:35] watchdog.front_door            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:25:35.295439246  [2023-08-14 02:25:35] ffmpeg.front_door.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:35.295442597  [2023-08-14 02:25:35] ffmpeg.front_door.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:35.295528993  [2023-08-14 02:25:35] ffmpeg.front_door.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:35.295530960  [2023-08-14 02:25:35] watchdog.front_yard            ERROR   : Ffmpeg process crashed unexpectedly for front_yard.
2023-08-14 02:25:35.295581576  [2023-08-14 02:25:35] watchdog.front_yard            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:25:35.295624595  [2023-08-14 02:25:35] ffmpeg.front_yard.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:35.295665715  [2023-08-14 02:25:35] ffmpeg.front_yard.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:35.295718087  [2023-08-14 02:25:35] ffmpeg.front_yard.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:35.304851186  [2023-08-14 02:25:35] ffmpeg.front_door.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:35.314708255  [2023-08-14 02:25:35] ffmpeg.front_yard.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:35.314722036  [2023-08-14 02:25:35] ffmpeg.front_yard.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:35.314727185  [2023-08-14 02:25:35] ffmpeg.front_yard.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:35.314738011  [2023-08-14 02:25:35] watchdog.front_yard            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:25:35.314745904  [2023-08-14 02:25:35] watchdog.front_yard            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:25:35.314750796  [2023-08-14 02:25:35] ffmpeg.front_door.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:35.314825360  [2023-08-14 02:25:35] ffmpeg.front_door.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:35.314830985  [2023-08-14 02:25:35] watchdog.front_door            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:25:35.314835320  [2023-08-14 02:25:35] watchdog.front_door            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:25:36.975190927  [2023-08-14 02:25:36] frigate.video                  ERROR   : front_door: Unable to read frames from ffmpeg process.
2023-08-14 02:25:36.975836799  [2023-08-14 02:25:36] frigate.video                  ERROR   : front_door: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:25:37.684684737  [2023-08-14 02:25:37] frigate.video                  ERROR   : front_yard: Unable to read frames from ffmpeg process.
2023-08-14 02:25:37.684898921  [2023-08-14 02:25:37] frigate.video                  ERROR   : front_yard: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:25:45.321803105  [2023-08-14 02:25:45] watchdog.front_yard            ERROR   : Ffmpeg process crashed unexpectedly for front_yard.
2023-08-14 02:25:45.327703559  [2023-08-14 02:25:45] watchdog.front_yard            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:25:45.328896956  [2023-08-14 02:25:45] ffmpeg.front_yard.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:45.329691692  [2023-08-14 02:25:45] ffmpeg.front_yard.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:45.330301027  [2023-08-14 02:25:45] ffmpeg.front_yard.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:45.339227121  [2023-08-14 02:25:45] ffmpeg.front_yard.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:45.348487175  [2023-08-14 02:25:45] watchdog.front_door            ERROR   : Ffmpeg process crashed unexpectedly for front_door.
2023-08-14 02:25:45.348499478  [2023-08-14 02:25:45] watchdog.front_door            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:25:45.348507707  [2023-08-14 02:25:45] ffmpeg.front_door.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:45.348512616  [2023-08-14 02:25:45] ffmpeg.front_door.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:45.348517364  [2023-08-14 02:25:45] ffmpeg.front_door.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:45.349680532  [2023-08-14 02:25:45] ffmpeg.front_yard.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:45.351807191  [2023-08-14 02:25:45] ffmpeg.front_yard.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:45.351820273  [2023-08-14 02:25:45] watchdog.front_yard            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:25:45.351825692  [2023-08-14 02:25:45] watchdog.front_yard            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:25:45.365036175  [2023-08-14 02:25:45] ffmpeg.front_door.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:45.373479613  [2023-08-14 02:25:45] ffmpeg.front_door.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:45.374137693  [2023-08-14 02:25:45] ffmpeg.front_door.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:45.374589300  [2023-08-14 02:25:45] watchdog.front_door            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:25:45.374924649  [2023-08-14 02:25:45] watchdog.front_door            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:25:46.981575140  [2023-08-14 02:25:46] frigate.video                  ERROR   : front_door: Unable to read frames from ffmpeg process.
2023-08-14 02:25:46.981586686  [2023-08-14 02:25:46] frigate.video                  ERROR   : front_door: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:25:47.685649754  [2023-08-14 02:25:47] frigate.video                  ERROR   : front_yard: Unable to read frames from ffmpeg process.
2023-08-14 02:25:47.686873494  [2023-08-14 02:25:47] frigate.video                  ERROR   : front_yard: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:25:55.358884968  [2023-08-14 02:25:55] watchdog.front_yard            ERROR   : Ffmpeg process crashed unexpectedly for front_yard.
2023-08-14 02:25:55.359016043  [2023-08-14 02:25:55] watchdog.front_yard            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:25:55.359160089  [2023-08-14 02:25:55] ffmpeg.front_yard.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:55.359354402  [2023-08-14 02:25:55] ffmpeg.front_yard.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:55.359433204  [2023-08-14 02:25:55] ffmpeg.front_yard.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:55.371565695  [2023-08-14 02:25:55] ffmpeg.front_yard.record       ERROR   : [h264 @ 0x55655b6c7400] Missing reference picture, default is 0
2023-08-14 02:25:55.371861269  [2023-08-14 02:25:55] ffmpeg.front_yard.record       ERROR   : [h264 @ 0x55655b6c7400] decode_slice_header error
2023-08-14 02:25:55.373019109  [2023-08-14 02:25:55] ffmpeg.front_yard.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:55.373030083  [2023-08-14 02:25:55] ffmpeg.front_yard.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:55.373033979  [2023-08-14 02:25:55] ffmpeg.front_yard.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:55.373037221  [2023-08-14 02:25:55] watchdog.front_yard            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:25:55.373861189  [2023-08-14 02:25:55] watchdog.front_yard            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:25:55.390510636  [2023-08-14 02:25:55] watchdog.front_door            ERROR   : Ffmpeg process crashed unexpectedly for front_door.
2023-08-14 02:25:55.391902933  [2023-08-14 02:25:55] watchdog.front_door            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:25:55.398519215  [2023-08-14 02:25:55] ffmpeg.front_door.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:55.399028029  [2023-08-14 02:25:55] ffmpeg.front_door.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:55.399224954  [2023-08-14 02:25:55] ffmpeg.front_door.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:55.410494598  [2023-08-14 02:25:55] ffmpeg.front_door.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:25:55.411149957  [2023-08-14 02:25:55] ffmpeg.front_door.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:25:55.412083013  [2023-08-14 02:25:55] ffmpeg.front_door.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:25:55.412947053  [2023-08-14 02:25:55] watchdog.front_door            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:25:55.413315565  [2023-08-14 02:25:55] watchdog.front_door            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:25:57.031618962  [2023-08-14 02:25:57] frigate.video                  ERROR   : front_door: Unable to read frames from ffmpeg process.
2023-08-14 02:25:57.032701217  [2023-08-14 02:25:57] frigate.video                  ERROR   : front_door: Unable to read frames from ffmpeg process.
2023-08-14 02:25:57.032996356  [2023-08-14 02:25:57] frigate.video                  ERROR   : front_door: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:25:57.793680313  [2023-08-14 02:25:57] frigate.video                  ERROR   : front_yard: Unable to read frames from ffmpeg process.
2023-08-14 02:25:57.793730577  [2023-08-14 02:25:57] frigate.video                  ERROR   : front_yard: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:26:05.401181375  [2023-08-14 02:26:05] watchdog.front_yard            ERROR   : Ffmpeg process crashed unexpectedly for front_yard.
2023-08-14 02:26:05.401819661  [2023-08-14 02:26:05] watchdog.front_yard            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:26:05.401830606  [2023-08-14 02:26:05] ffmpeg.front_yard.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:05.401857341  [2023-08-14 02:26:05] ffmpeg.front_yard.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:05.402034200  [2023-08-14 02:26:05] ffmpeg.front_yard.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:05.431754385  [2023-08-14 02:26:05] watchdog.front_door            ERROR   : Ffmpeg process crashed unexpectedly for front_door.
2023-08-14 02:26:05.436150702  [2023-08-14 02:26:05] watchdog.front_door            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:26:05.443331550  [2023-08-14 02:26:05] ffmpeg.front_door.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:05.443343578  [2023-08-14 02:26:05] ffmpeg.front_door.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:05.443349200  [2023-08-14 02:26:05] ffmpeg.front_door.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:05.443356844  [2023-08-14 02:26:05] ffmpeg.front_yard.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:05.443362206  [2023-08-14 02:26:05] ffmpeg.front_yard.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:05.443366783  [2023-08-14 02:26:05] ffmpeg.front_yard.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:05.443371094  [2023-08-14 02:26:05] watchdog.front_yard            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:26:05.443375701  [2023-08-14 02:26:05] watchdog.front_yard            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:26:05.454902011  [2023-08-14 02:26:05] ffmpeg.front_door.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:05.456021059  [2023-08-14 02:26:05] ffmpeg.front_door.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:05.456927052  [2023-08-14 02:26:05] ffmpeg.front_door.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:05.457874303  [2023-08-14 02:26:05] watchdog.front_door            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:26:05.458713840  [2023-08-14 02:26:05] watchdog.front_door            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:26:07.109154522  [2023-08-14 02:26:07] frigate.video                  ERROR   : front_door: Unable to read frames from ffmpeg process.
2023-08-14 02:26:07.109619386  [2023-08-14 02:26:07] frigate.video                  ERROR   : front_door: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:26:07.792303589  [2023-08-14 02:26:07] frigate.video                  ERROR   : front_yard: Unable to read frames from ffmpeg process.
2023-08-14 02:26:07.793165327  [2023-08-14 02:26:07] frigate.video                  ERROR   : front_yard: ffmpeg process is not running. exiting capture thread...
2023-08-14 02:26:15.455485066  [2023-08-14 02:26:15] watchdog.front_yard            ERROR   : Ffmpeg process crashed unexpectedly for front_yard.
2023-08-14 02:26:15.455496977  [2023-08-14 02:26:15] watchdog.front_yard            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:26:15.455503807  [2023-08-14 02:26:15] ffmpeg.front_yard.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:15.455507706  [2023-08-14 02:26:15] ffmpeg.front_yard.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:15.455511654  [2023-08-14 02:26:15] ffmpeg.front_yard.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:15.476206213  [2023-08-14 02:26:15] ffmpeg.front_yard.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:15.476856296  [2023-08-14 02:26:15] ffmpeg.front_yard.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:15.477354004  [2023-08-14 02:26:15] ffmpeg.front_yard.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:15.477868336  [2023-08-14 02:26:15] watchdog.front_yard            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:26:15.478140129  [2023-08-14 02:26:15] watchdog.front_yard            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:26:15.487052196  [2023-08-14 02:26:15] watchdog.front_door            ERROR   : Ffmpeg process crashed unexpectedly for front_door.
2023-08-14 02:26:15.487558872  [2023-08-14 02:26:15] watchdog.front_door            ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-08-14 02:26:15.487900990  [2023-08-14 02:26:15] ffmpeg.front_door.detect       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:15.488147143  [2023-08-14 02:26:15] ffmpeg.front_door.detect       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:15.488413146  [2023-08-14 02:26:15] ffmpeg.front_door.detect       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:15.504945956  [2023-08-14 02:26:15] ffmpeg.front_door.record       ERROR   : Option hwaccel (use HW accelerated decoding) cannot be applied to output url vaapi -- you are trying to apply an input option to an output file or vice versa. Move this option before the file it belongs to.
2023-08-14 02:26:15.504953323  [2023-08-14 02:26:15] ffmpeg.front_door.record       ERROR   : Error parsing options for output file vaapi.
2023-08-14 02:26:15.504956630  [2023-08-14 02:26:15] ffmpeg.front_door.record       ERROR   : Error opening output files: Invalid argument
2023-08-14 02:26:15.504959498  [2023-08-14 02:26:15] watchdog.front_door            INFO    : Terminating the existing ffmpeg process...
2023-08-14 02:26:15.504962372  [2023-08-14 02:26:15] watchdog.front_door            INFO    : Waiting for ffmpeg to exit gracefully...
2023-08-14 02:26:17.178968772  [2023-08-14 02:26:17] frigate.video                  ERROR   : front_door: Unable to read frames from ffmpeg process.
2023-08-14 02:26:17.179209151  [2023-08-14 02:26:17] frigate.video                  ERROR   : front_door: ffmpeg process is not running. exiting capture thread...
```

### FFprobe output from your camera

```shell
-
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

SV3C Cloud

**Comment by NickM-27** [comment]
Frigate is compatible with the n100, i965 should not be used with this CPU

**Comment by NickM-27** [comment]
Also, please run vaapi on the host and inside frigate (without i965) and show both results.

**Comment by AlekseiLitvin** [comment]
@NickM-27 Can explain how can I `run vaapi`? Do you mean to set ` hwaccel_args: preset-vaapi` in the configuration?

**Comment by NickM-27** [comment]
My bad, I mean `vainfo`

**Comment by AlekseiLitvin** [comment]
`vainfo` from the container:
```
error: XDG_RUNTIME_DIR is invalid or not set in the environment.
error: can't connect to X server!
error: failed to initialize display
```

on the host:
```
error: can't connect to X server!
error: failed to initialize display
```

**Comment by NickM-27** [comment]
Looks like there's the problem, host is missing drivers.

**Comment by AlekseiLitvin** [comment]
@NickM-27 I got the same results on my second PC, which is based on  Intel Celeron N5105, and Frigate works perfectly fine here.
For the context: I am using Headless Ubuntu Server

**Comment by NickM-27** [comment]
Are you including the entire output from the command? Otherwise it'll show the supported profiles.

**Comment by AlekseiLitvin** [comment]
That is the output of vainfo on my second PC, inside the frigate container. Can you suggest, what drivers do I need? I followed exactly the same OS installation process on both machines
```
VAINFO OUTPUT
Return Code: 0

Process Output:

vainfo: VA-API version: 1.17 (libva 2.10.0)
vainfo: Driver version: Intel iHD driver for Intel(R) Gen Graphics - 23.1.1 ()
vainfo: Supported profile and entrypoints
VAProfileNone : VAEntrypointVideoProc
VAProfileNone : VAEntrypointStats
VAProfileMPEG2Simple : VAEntrypointVLD
VAProfileMPEG2Main : VAEntrypointVLD
VAProfileH264Main : VAEntrypointVLD
VAProfileH264Main : VAEntrypointEncSliceLP
VAProfileH264High : VAEntrypointVLD
VAProfileH264High : VAEntrypointEncSliceLP
VAProfileVC1Simple : VAEntrypointVLD
VAProfileVC1Main : VAEntrypointVLD
VAProfileVC1Advanced : VAEntrypointVLD
VAProfileJPEGBaseline : VAEntrypointVLD
VAProfileJPEGBaseline : VAEntrypointEncPicture
VAProfileH264ConstrainedBaseline: VAEntrypointVLD
VAProfileH264ConstrainedBaseline: VAEntrypointEncSliceLP
VAProfileVP8Version0_3 : VAEntrypointVLD
VAProfileHEVCMain : VAEntrypointVLD
VAProfileHEVCMain : VAEntrypointEncSliceLP
VAProfileHEVCMain10 : VAEntrypointVLD
VAProfileHEVCMain10 : VAEntrypointEncSliceLP
VAProfileVP9Profile0 : VAEntrypointVLD
VAProfileVP9Profile1 : VAEntrypointVLD
VAProfileVP9Profile2 : VAEntrypointVLD
VAProfileVP9Profile3 : VAEntrypointVLD
VAProfileHEVCMain422_10 : VAEntrypointVLD
VAProfileHEVCMain444 : VAEntrypointVLD
VAProfileHEVCMain444 : VAEntrypointEncSliceLP
VAProfileHEVCMain444_10 : VAEntrypointVLD
VAProfileHEVCMain444_10 : VAEntrypointEncSliceLP
```

**Comment by NickM-27** [comment]
Need iHD drivers

**Comment by AlekseiLitvin** [comment]
I found a solution to the problem:

It turns out drivers for Intel N100 are not available for Ubuntu Server 22.04, I updated it to 23.04 and the problem was fixed.

Thank you @NickM-27 , I am closing the issue
